lib/newrelic_httprb.rb

```diff
@@ -0,0 +1,2 @@
+require "newrelic_httprb/instrumentation"
+require "newrelic_httprb/version"
```

Comment:
Seems like it got fixed in 60b61cbe9156646dfc0967662f03e3046867e441 😸 

Yes yes, I'm just here because I missed reviewing your code guys 😝 ❤️